packages/components/src/components/dropdown/_dropdown.scss

```diff
@@ -65,6 +65,13 @@
     }
   }
 
+  // Menu's triggering element updated to button with Downshift v5 upgrade
+  .#{prefix}--dropdown button {
+    @include button-reset;
+    text-align: left;
+    padding: 0 rem(48px) 0 rem(16px);
+  }
```

Comment:
Inline dropdown not rendering properly 

<img width="331" alt="Screen Shot 2020-04-02 at 12 32 54 PM" src="https://user-images.githubusercontent.com/11928039/78294895-fbd83100-74df-11ea-9c9a-c298bb82fc96.png">